applications/FemToDemApplication/python_scripts/CouplingFemDem.py

```diff
@@ -578,16 +571,16 @@ def CreateInitialSkinDEM(self):
 #============================================================================================================================
 
     def InitializeIntegrationPointsVariables(self):
-
+        elements = self.FEM_Solution.main_model_part.Elements
         utils = KratosMultiphysics.VariableUtils()
-        utils.SetNonHistoricalVariable(KratosFemDem.STRESS_THRESHOLD, 0.0, self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.DAMAGE_ELEMENT, 0.0, self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.PRESSURE_EXPANDED, 0, self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.IS_SKIN, 0, self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.SMOOTHING, 0, self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.STRESS_VECTOR, [0.0,0.0,0.0], self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.STRAIN_VECTOR, [0.0,0.0,0.0], self.FEM_Solution.main_model_part.Elements)
-        utils.SetNonHistoricalVariable(KratosFemDem.STRESS_VECTOR_INTEGRATED, [0.0,0.0,0.0], self.FEM_Solution.main_model_part.Elements)
+        utils.SetNonHistoricalVariable(KratosFemDem.STRESS_THRESHOLD, 0.0, elements)
```

Comment:
There is a new specific method o set zero values, but we can live with this for now

Reply:
which is? just for curiosity

Reply:
SetNonHistoricalVariableToZero and SetHistoricalVariableToZero

Reply:
Be careful with the Vector, I don't know what it does, because doesn't know the size

Reply:
A bounded array should not be a problem